Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/README.MD

```diff
@@ -0,0 +1,84 @@
+# Google.Apis.Auth.AspNetCore3 sample application.
+
+The [Google.Apis.Auth.AspNetCore3][package] contains a Google-specific
+OpenIdConnect auth handler.
+It supports incremental auth, and defines an injectable IGoogleAuthProvider
+to supply Google credentials.
+
+Google.Apis.Auth.AspNetCore3.IntegrationTests, is a standard ASP.NET Core 3
+application that shows how to configure and use the main features of
+[Google.Apis.Auth.AspNetCore3][package].
+
+## GCP setup prerequisites.
+
+### GCP project.
+
+First you need to create a Google Cloud project if you don't have one already.
+Follow [these instructions][new-gcp-project] to do so. This will be the project
+the app is identified with.
+
+### Enabling APIs used by this app.
+
+The app uses the [Storage][storage-api] and [Translate][translate-api] APIs.
+These need to be enabled in your GCP project. To enable the APIs, follow
+[these instructions][enable-apis].
+
+### Create authorization credentials.
+
+These credentials are the ones that identify your app to Google. Follow
+[these instructions][oauth-creds] to create authorization credentials and download
+the client_secrets.json file. Two highlights:
+
+  - Notice that the credentials' type must be *Web application*.
+  - For running this app, the only redirect URI you need to add is
+  https://localhost:5001/signin-oidc
+
+### Set up environment variables.
+
+- Set `TEST_WEB_CLIENT_SECRET_FILENAME` to the path of the client_secrets.json
+file you downloaded on the previous step.
+- Set `GOOGLE_PROJECT_ID` to the GCP Project ID of the GCP project you are using.
+  - As part of this app, we'll list the Storage buckets that the authenticated user
+  has access to on the GCP project with ID in `GOOGLE_PROJECT_ID`. This project doesn't
+  have to be the same as the project you created the credentials in. For instance,
+  for a real world app, you may have a GCP project that only your organization can access,
+  and that's were you'd create the credentials for the app. You then could have a second
+  GCP project that users of your app can access, and there they would create their Storage
+  buckets or other resources, either through your app or directly throught the Google Cloud Console.
```

Comment:
```suggestion
- Set `TEST_WEB_CLIENT_SECRET_FILENAME` to the path of the client_secrets.json
  file you downloaded on the previous step.
- Set `GOOGLE_PROJECT_ID` to the GCP Project ID of the GCP project you are using.
  - As part of this app, we'll list the Storage buckets that the authenticated user
    has access to on the GCP project with ID in `GOOGLE_PROJECT_ID`. This project doesn't
    have to be the same as the project you created the credentials in. For instance,
    for a real world app, you may have a GCP project that only your organization can access,
    and that's were you'd create the credentials for the app. You then could have a second
    GCP project that users of your app can access, and there they would create their Storage
    buckets or other resources, either through your app or directly throught the Google Cloud Console.
```

(Just to make the indentation level of all *text* consistent on a per-bullet level.)